firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FirebaseAppDistributionLifecycleNotifier.java

```diff
@@ -181,6 +225,28 @@ public void onResumed(Activity activity) {
     }
   }
 
+  @Nullable
+  private static <A extends Activity> Activity getActivityWithIgnoredClass(
+      Activity activity, @Nullable Activity fallbackActivity, @Nullable Class<A> classToIgnore) {
+    if (classToIgnore != null && classToIgnore.isInstance(activity)) {
+      return fallbackActivity;
+    } else {
+      return activity;
+    }
+  }
+
+  private void updateCurrentActivity(@Nullable Activity activity) {
+    synchronized (lock) {
+      if (currentActivity != activity) {
+        if (currentActivity != null) {
+          // Store a reference to the previous activity in case the current activity is ignored
+          previousActivity = currentActivity;
```

Comment:
If the current activity is an ignored one, do you even need to store it?

Reply:
I should make this comment clearer - what I mean is that the currentActivity _may be ignored later_ when `applyToNullableForegroundActivity(classToIgnore, ...)` is called.